### ISSUE TITLE
Improve filesystem build scripts

### ISSUE THREAD

**Issue by maxonthegit** [maxonthegit/netkit-uml-filesystem#1]
Filesystem build scripts are currently designed to create a single filesystem image. If a user wants to create an alternative image, she has to remove (or rename/move elsewhere) the existing one before initiating a new build process.
The scripts could be improved in order to make creation of different filesystem images easier.

Also check `$(<file)` statements in the filesystem's `Makefile.devel`: they do not seem to work.

**Comment by kartoch** [comment]
I think the simplest idea is the following: `$NETKIT_HOME/fs/` contains all fs files, with a link pointing to the default fs. It also contains all filesystems builds, each of one in a directory (for instance `$NETKIT_HOME/fs/netkit-uml-filesystem-build/`, `$NETKIT_HOME/fs/netkit-uml-lenny-filesystem-build/`, ...). What do you think ?

**Comment by maxonthegit** [comment]
Yes, that looks pretty straightforward and effective. Indeed, that was the initial idea that motivated the fact that `netkit-fs` is currently a symbolic link to the actual filesystem image.
